src/Http/Livewire/FileList.php

```diff
@@ -39,6 +39,8 @@ public function deleteFile(string $fileName)
             $this->selectedFileName = '';
             $this->emit('fileSelected', $this->selectedFileName);
         }
+        
+        LogViewer::getFiles()->each->clearIndexCache();
```

Comment:
Although technically it works, it does a little too much. If you have 10 log files all cached, it doesn't make sense to clear the cache of all of them when just **one** file gets deleted.

Have a look at the `LogFile` class and notice it has a `clearIndexCache()` method. What about calling it inside the `delete()` method?

Reply:
@arukompas, I've just implemented that.